pystac/layout.py

```diff
@@ -20,6 +20,9 @@ class TemplateError(Exception):
     string into data for :class:`~pystac.layout.LayoutTemplate`
     """
 
+    # This class is deprecated by its duplicate in `errors.py`. It is retained
+    # here for backwards compatibility. It will be removed in pystac version 2.0.0.
+
     pass
```

Comment:
I think it would be appropriate to add a `DeprecationWarning` to `__init__`?

Reply:
Done. Also updated the docstring with a deprecated warning.